css/dashboard.css

```diff
@@ -0,0 +1,137 @@
+*, *::after, *::before {
```

Comment:
Why do I see css again?

Reply:
All the css files have been removed in the future PRs. Only single styles.scss is used which is converted to styles.css ultimately.